Automattic/wp-calypso
# Add anchor to quicklinks

#### Changes proposed in this Pull Request

* In quicklinks, add a link to anchor.fm

#### Testing instructions



* Visit calypso homepage
* Check quicklinks for a link to anchor.fm

![2021-03-03_13-33](https://user-images.githubusercontent.com/937354/109861714-3a60f900-7c25-11eb-9f09-9f509ccbe9d3.png)

![2021-03-03_13-34](https://user-images.githubusercontent.com/937354/109861759-451b8e00-7c25-11eb-81fe-812074c82276.png)

Related to 514-gh-Automattic/dotcom-manage
